.github/workflows/ci-build.yml

```
@@ -36,6 +36,31 @@ jobs:
       - run: yarn install --no-lockfile
       - run: yarn test:ember
 
+  types-scenarios:
+    name: "Types: ${{ matrix.tsVersion }}"
+    runs-on: ubuntu-latest
+    needs: test
+
+    strategy:
+      matrix:
+        tsVersion:
+          - 4.4
+          - 4.6
+          - 4.8
+          - 5.0
+          - 5.1
```

This is a really weird set of versions. 😂 Let's go with 4.9+ instead:

```suggestion
          - 4.9
          - 5.0
          - 5.1
```

---

done! 🥳